ignite/metrics/precision.py

```diff
@@ -36,4 +47,11 @@ def update(self, output):
     def compute(self):
         if self._all_positives is None:
             raise NotComputableError('Precision must have at least one example before it can be computed')
-        return self._true_positives / self._all_positives
+        elif self._all_positives.eq(0.0).any():
+            warnings.warn('Labels with no predicted examples are set to have precision of 0.0.', UndefinedMetricWarning)
```

Comment:
thought: this might get annoying in the users output. Wdyt?

Reply:
Yea I can definitely see that. Do you think we maybe should just skip the warning?

Reply:
Yeah, or can log it with level WARNING

Reply:
I went ahead and removed it.